docs/gaia/validators/validator-setup.md

```diff
@@ -22,32 +60,63 @@ Your `cosmosvalconspub` can be used to create a new validator by staking tokens.
 gaiad tendermint show-validator
 ```
 
-Next, craft your `gaiacli tx stake create-validator` command:
+Next, craft your `gaiad gentx` command:
 
 ::: warning Note
 Don't use more `steak` thank you have! You can always get more by using the [Faucet](https://faucetcosmos.network/)!
 :::
 
 ```bash
-gaiacli tx stake create-validator \
-  --amount=5steak \
-  --pubkey=$(gaiad tendermint show-validator) \
-  --moniker="choose a moniker" \
-  --chain-id=<chain_id> \
-  --from=<key_name> \
-  --commission-rate="0.10" \
-  --commission-max-rate="0.20" \
-  --commission-max-change-rate="0.01"
+gaiad gentx \
+  --amount <amount_of_delegation> \
+  --commission-rate <commission_rate> \
+  --commission-max-rate <commission_max_rate> \
+  --commission-max-change-rate <commission_max_change_rate> \
+  --pubkey <consensus_pubkey> \
+  --name <key_name>
 ```
 
 __Note__: When specifying commission parameters, the `commission-max-change-rate`
 is used to measure % _point_ change over the `commission-rate`. E.g. 1% to 2% is
 a 100% rate increase, but only 1 percentage point.
 
+__Note__: If unspecified, `consensus_pubkey` will default to the output of `gaiad tendermint show-validator`.
+`key_name` is the name of the private key that will be used to sign the transaction.
+
 ::: tip
-Use `gaiacli tx stake create-validator -h` to get a list of all the available flags.
+Consult `gaiad gentx --help` for more information on the flags defaults.
 :::
 
+#### What are genesis transactions for?
+
+A `genesis transaction` is a JSON file carrying a self-delegation. All genesis transactions are collected by a `genesis coordinator` and validated against an initial `genesis.json`. Such initial `genesis.json` contains only a list of accounts and their coins. Once the transactions are processed, they are merged in the `genesis.json`'s `gentxs` field.
```

Comment:
Can't you generate a `gentx` from any account to a validator? Does it really need to be a self-delegation? (in other words, can you specific --validator in `gentx`?)

Reply:
Not in `gentx`. You can do it with the `gaiacli tx create-validator` command though

Reply:
I'm confused. If we're talking about self-delegation, this means that the address belongs to a validator. At genesis, we'll have a suggested Atom allocation. Atom will be allocated to fundraiser participants. We want these participants to be able to generate a delegation tx that points to a validator address. But the validator address will most likely be different from the fundraiser address. Even if a validator operator participated in the fundraiser, it would be bad security practice to use the fundraiser address as validator address. I'm lacking clarity on what the exact process looks like for this.

Reply:
`gaiad gentx` is only for self-delegation. `gaiacli tx create-validator` can be used for both.

Reply:
So let's say I have Atoms on `address A` and want to delegate to a validator address `address B` and have this delegation be processed during startup (similar to a `gentx`), how would I do that?

Reply:
You will have to use the `gaiacli tx stake create-validator` command